templates/usr/local/bin/rsync_debian_mirror.j2

```diff
@@ -73,6 +75,12 @@ function writetracefile() {
 
     mv "${base_path}/${trace}.new" "${base_path}/${trace}"
 
+    echo "$archive_version" >> ${base_path}/${trace}
+
+    if grep -q "pkg.adfinis.com" ${base_path}/project/trace/_traces; then
+           echo "pkg.adfinis.com" >> ${base_path}/project/trace/_traces
+    fi
```

Comment:
I imported this from the productive script. For me, this makes not really sense.
It should be a negative check, shouldnt it?

Reply:
Indeed, this seems backwards. It should probably be `if ! grep -q …`.

Reply:
Resolved in d1f7d4ec54abffe6f9c7bd78a171078e6cd61b0e